controllers/profiles.js

```diff
@@ -0,0 +1,129 @@
+const fs = require("fs")
+const profiles = require("../data/profiles.json")
+
+const errorResHelper = (errorDescription, res) => {
+  res.status(400).send(
+    JSON.stringify({
+      error: errorDescription,
+    })
+  )
+}
+
+const createProfile = (req, res) => {
+  const formattedNow2ISO = new Date()
+    .toISOString()
+    .split("T")
+    .join(" ")
+    .split(".")[0]
+  const missing = []
+  const newProfile = {
+    idProfile: new Date().getTime().toString(),
+    name: req.query.name ?? missing.push("'name'"),
+    created: {
+      admin: req.query.idAdmin ?? missing.push("'idAdmin'"),
+      timestamp: formattedNow2ISO,
+    },
+    modified: [],
```

Comment:
it is common to save the last modification date instead of an array of modification history.